programs/eosioc/main.cpp

```diff
@@ -436,6 +436,25 @@ struct set_action_permission_subcommand {
    }
 };
 
+struct currency_stats {
+   asset supply;
+};
+
+FC_REFLECT(currency_stats, (supply));
+
```

Comment:
For everything else in eosioc main the type is defined in the api plugin. Seems like prompting this to a full fledge understood json of the eosiod api would be appropriate. In this case chain_plugin.